pkg/cmd/component/component/component.go

```diff
@@ -0,0 +1,32 @@
+package component
```

Comment:
Also run `kam component add` on its own, and it should not put you in interactive mode.  Compare this with `kam environment add`

Reply:
If we see the behavior of the old `bootstrap command` when we execute `kam bootstrap` it will initiate the interactive mode. Similarly the `kam bootstrap-new` will ask for the interactive mode. 
Do we still require to change this behaviour?

Reply:
Good point. The original `environment add ` doesn't have interactive mode.